zuul-core/src/main/java/com/netflix/zuul/netty/server/ClientRequestReceiver.java

```diff
@@ -141,6 +140,9 @@ public void channelRead(final ChannelHandlerContext ctx, Object msg) throws Exce
                 zuulRequest.getContext().setShouldSendErrorResponse(true);
             }
 
+            handleExpect100Continue(ctx, clientRequest);
```

Comment:
I moved this down below the error handling.   I *think* this is correct, but I may be mistaken.